README.md

```diff
@@ -18,6 +18,17 @@ npm install graphql-constraint-directive@v2
 ```
 
 ## Usage
+
+### Schema wrapper
+
+Implementation based on schema wrappers - basic scalars are wrapped as custom scalars with validations. 
+
+Pros:
+* posibility to validate output
```

Comment:
@velias what do you mean by output in this context? Can this be elaborated?

Reply:
If I understand it correctly, that wrapper implementation is capable to validate output values returned from GraphQL query - directive is allowed at FIELD_DEFINITION.
Or did I misunderstood something?

Reply:
@velias can a test case be added to confirm? Would also be great to explain this further with an example as it wasn't clear to me what this meant

Reply:
But it is not feature added in this PR, schema wrapper is your implementation :-)

Reply:
OK, in reality there are tests for this feature, in `int.test.js` and `float.test.js` and `string.test.js`

Reply:
I improved wording to be much clearer (I hope)